packages/vuetify/src/labs/VPie/VPie.tsx

```diff
@@ -0,0 +1,378 @@
+      return visibleItemsKeys.value.includes(item.key)
+    }
+
+    function toggle (item: PieItem) {
+      if (isActive(item)) {
+        visibleItemsKeys.value = visibleItemsKeys.value.filter(x => x !== item.key)
+      } else {
+        visibleItemsKeys.value = [...visibleItemsKeys.value, item.key]
+      }
+    }
+
+    const tooltipItem = shallowRef<PieItem | null>(null)
+    const tooltipVisible = shallowRef(false)
+
+    let mouseLeaveTimeout = null! as ReturnType<typeof setTimeout>
+
+    function onMouseenter (item: PieItem) {
+      if (!props.tooltip) return
+
+      clearTimeout(mouseLeaveTimeout)
+      tooltipVisible.value = true
+      tooltipItem.value = item
+    }
+
+    function onMouseleave () {
+      if (!props.tooltip) return
+
+      clearTimeout(mouseLeaveTimeout)
+      mouseLeaveTimeout = setTimeout(() => {
+        tooltipVisible.value = false
+
+        // intentionally reusing timeout here
+        mouseLeaveTimeout = setTimeout(() => {
+          tooltipItem.value = null
+        }, 500)
+      }, 100)
+    }
+
+    return () => {
+      const itemProps = VPieSegment.filterProps(pick(props, [
+        'animation',
+        'gap',
+        'rounded',
+        'hideSlice',
+        'reveal',
+      ]))
+
+      const defaultTooltipTransition = {
+        name: 'fade-transition',
+        duration: 150,
+      }
+
+      const tooltipProps = {
+        item: tooltipItem.value,
+        modelValue: tooltipVisible.value,
+        titleFormat: typeof props.tooltip === 'object' ? props.tooltip.titleFormat : '[title]',
+        subtitleFormat: typeof props.tooltip === 'object' ? props.tooltip.subtitleFormat : '[value]',
+        transition: typeof props.tooltip === 'object' ? props.tooltip.transition : defaultTooltipTransition,
+        offset: typeof props.tooltip === 'object' ? props.tooltip.offset : 16,
+      }
+
+      const legendDefaults = {
+        VChipGroup: {
+          direction: legendDirection.value,
+        },
+        VChip: {
+          density: props.density,
+        },
+        VAvatar: {
+          size: legendAvatarSize.value,
+        },
+      }
+
+      const tooltipDefaults = {
+        VAvatar: {
+          size: 28,
+        },
+      }
+
+      const avatarSlot = ({ item }: { item: PieItem }) => (
+        <VAvatar color={ item.color } start>
+          { item.pattern && (
+            <svg height="40" width="40">
+              <rect width="40" height="40" fill={ item.pattern } />
+            </svg>
+          )}
+        </VAvatar>
+      )
+
+      return (
+        <div
+          class={[
+            'v-pie',
+            `v-pie--legend-${legendMode.value}`,
+          ]}
+          style={{
+            '--v-pie-size': convertToUnit(props.size),
+          }}
+        >
+          { slots.title?.() ?? (props.title && (<div class="v-pie__title">{ props.title }</div>)) }
+          <div
+            class={[
+              'v-pie__content',
+              ...textColorClasses.value,
```

Comment:
```suggestion
              textColorClasses.value,
```

Reply:
Actually, this `textColorClasses` was always empty. Fixed by using background class and overriding background to none.